Formula/s/sing-box.rb

```diff
@@ -26,7 +26,7 @@ def install
   end
 
   service do
-    run [opt_bin/"sing-box", "run", "--config", etc/"sing-box/config.json"]
+    run [opt_bin/"sing-box", "run", "--config", etc/"sing-box/config.json", "--directory", etc/"sing-box"]
```

Comment:
May be better to set `--directory` as subdirectory of `var` rather than `etc`.

Upstream's systemd service as an example https://github.com/SagerNet/sing-box/blob/dev-next/release/config/sing-box.service#L9
```service
ExecStart=/usr/bin/sing-box -D /var/lib/sing-box -C /etc/sing-box run
```

It looks like they define 2 separate directories for these to align to common Linux directory structure.